app/scripts/modules/core/src/pipeline/config/templates/v2/configurePipelineTemplateModalV2.controller.ts

```diff
@@ -115,19 +112,14 @@ export class ConfigurePipelineTemplateModalV2Controller implements IController {
     return PipelineTemplateReader.getPipelinePlan(config)
       .then(plan => {
         const { parameterConfig, notifications, triggers, expectedArtifacts } = plan;
-        const {
-          inheritTemplateParameters,
-          inheritTemplateNotifications,
-          inheritTemplateTriggers,
-          inheritTemplateExpectedArtifacts,
-        } = this.state;
+        const { inheritTemplateParameters, inheritTemplateNotifications, inheritTemplateTriggers } = this.state;
 
         const configWithInheritedValues = {
           ...config,
           ...(inheritTemplateParameters && parameterConfig ? { parameterConfig } : {}),
           ...(inheritTemplateNotifications && notifications ? { notifications } : {}),
           ...(inheritTemplateTriggers && triggers ? { triggers } : {}),
-          ...(inheritTemplateExpectedArtifacts && expectedArtifacts ? { expectedArtifacts } : {}),
+          ...(expectedArtifacts ? { expectedArtifacts } : {}),
```

Comment:
As of this [PR](https://github.com/spinnaker/orca/commit/ca1a5269bf20ad5807dcadce20611fedeb6202f8), inheritance of expectedArtifacts changed from opt in to always inherited. Currently only parameters, triggers, and notifications can be marked for exclusion.